src/test/java/com/herokuapp/theinternet/test/ForgotPasswordTest.java

```diff
@@ -0,0 +1,220 @@
+package com.herokuapp.theinternet.test;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.remote.DesiredCapabilities;
+/*import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;*/
+
+import java.util.concurrent.TimeUnit;
+
+public class ForgotPasswordTest {
+    private WebDriver driver;
+
+
+    @BeforeClass
+    public static void setupClass() {
+        WebDriverManager.chromedriver().setup();
+    }
+
+    @Test
+    public void retrievePassword() {
+        ChromeOptions chromeOptions = new ChromeOptions();
+        chromeOptions.addArguments("start-maximized");
+        driver = new ChromeDriver(chromeOptions);
+        driver.get("http://the-internet.herokuapp.com/forgot_password");
+        WebElement emailInput = driver.findElement(By.id("email"));
+        emailInput.clear();
+        emailInput.sendKeys("test@test.com");
+        emailInput.submit();
+
+    }
+
+    @Test
+    public void dragAndDropTest() {
+        ChromeOptions chromeOptions = new ChromeOptions();
+        chromeOptions.addArguments("start-maximized");
+        driver = new ChromeDriver(chromeOptions);
+        driver.get("http://the-internet.herokuapp.com/drag_and_drop");
+        WebElement columnAElement = driver.findElement(By.id("column-a"));
+        WebElement columnBElement = driver.findElement(By.id("column-b"));
+
+        Actions actions = new Actions(driver);
+        actions
+                .moveToElement(columnAElement)
+                .clickAndHold()
+                .moveToElement(columnBElement)
+                .release()
+                .build()
+                .perform();
+
+    }
+
+    @Test
+    public void addRemoveElementsTest() {
+        ChromeOptions chromeOptions = new ChromeOptions();
+        chromeOptions.addArguments("start-maximized");
+        driver = new ChromeDriver(chromeOptions);
+        driver.get("http://the-internet.herokuapp.com/add_remove_elements/");
+        WebElement addElement = driver.findElement(By.xpath("//*[@id=\"content\"]/div/button"));
+        addElement.click();
+        driver.manage().timeouts().implicitlyWait(3, TimeUnit.SECONDS);
```

Comment:
this is configuration, not waiter. This line does not waits, it just sets implicit timeout for webdriver